app/jobs/turbo/streams/broadcast_job.rb

```diff
@@ -1,6 +1,8 @@
 # The job that powers the <tt>broadcast_render_later_to</tt> available in <tt>Turbo::Streams::Broadcasts</tt> for rendering
 # turbo stream templates.
 class Turbo::Streams::BroadcastJob < ActiveJob::Base
+  discard_on ActiveJob::DeserializationError
```

Comment:
Could we add tests to cover this behavior?

Maybe something that creates a record, enqueues a job, [performs the jobs](https://edgeapi.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-perform_enqueued_jobs), then asserts that [the job was only run once](https://edgeapi.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-assert_performed_jobs) and that another is [not enqueued](https://edgeapi.rubyonrails.org/classes/ActiveJob/TestHelper.html#method-i-assert_enqueued_jobs).